packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -214,7 +214,7 @@ function FontCollection( { slug } ) {
 		}
 
 		try {
-			await installFont( fontFamily );
+			await installFonts( [ fontFamily ] );
```

Comment:
```suggestion
			await installFonts( fontFamily );
```

Only a small nitpick but I think I'd rather the `installFonts` function handled converting arguments to an array (e.g. checking if `fontFamiliesToInstall` is an array, if not then converting it), as I think it would slightly improve the readability here.

Reply:
I'm generally opposed to passing different types of arguments into a function.  If we need to change how that value is passed to offer clarity I would rather do that (i.e. `fontFamiliesToInstall = []; fontFamiliesToInstall.push( fontFamily ); installFonts( fontFamiliesToInstall ); `

Reply:
Yeah I think that's a good alternative too 👍